core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -518,8 +503,6 @@
       "The maximum number of threads to use to sort logs during" + " recovery"),
   TSERV_SORT_BUFFER_SIZE("tserver.sort.buffer.size", "10%", PropertyType.MEMORY,
       "The amount of memory to use when sorting logs during recovery."),
-  TSERV_ARCHIVE_WALOGS("tserver.archive.walogs", "false", PropertyType.BOOLEAN,
```

Comment:
It's a bit weird that this one isn't used. @keith-turner might be interested, since he's been looking a lot at the WALs lately.

Reply:
If this one should be added back in, I will have to also add it back to HOT_PATH_PROPERTIES. Leaving this comment as a note to myself to look at that

Reply:
I think its ok to remove this property for now.  The prop may need a different name anyway, as it would now be a GC responsibility.  I opened #459